tsconfig.paths.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": ".",
     "paths": {
       "types": [
-        "../../back-end/megak-headhunter-6-backend/types"
+        "../megak-headhunter-6-backend/types"
```

Comment:
You shouldn't commit all the files you have changed. This file shouldn't be here!